templates/repo/projects/view.tmpl

```diff
@@ -84,7 +84,7 @@
 				<div class="board-column-header df ac sb">
 					<div class="ui large label board-label py-2">
 						<div class="ui small circular grey label board-card-cnt">
-							{{len .Issues}}
+							{{.NumIssues}}
```

Comment:
did result in a wrong number for the "unassigned" board

Reply:
Please review the function `NumIssues` again.